gateway/src/apicast/policy/routing/README.md

```diff
@@ -340,6 +340,35 @@ the path is not `/accounts`:
     }
   }
 ```
+## Liquid templating on match
+
+It is possible to use liquid templating in the match section. This allows to
+define rules with dynamic information and provide a way to match with any
+request info.
```

Comment:
I think we should be more specific in this description. I think we should mention what information of the `original_request` is available, and also, the variables that can be accessed because they're in `ngx.var`. Should we link https://nginx.org/en/docs/varindex.html (if we expose all of them) ?